internal/postrs/proof.go

```diff
@@ -20,6 +19,26 @@ import (
 
 type ScryptParams = C.ScryptParams
 
+type Config struct {
+	MinNumUnits   uint32
+	MaxNumUnits   uint32
+	LabelsPerUnit uint64
+
+	K1 uint32 // K1 specifies the difficulty for a label to be a candidate for a proof.
+	K2 uint32 // K2 is the number of labels below the required difficulty required for a proof.
+	K3 uint32 // K3 is the size of the subset of proof indices that is validated.
+
+	PowDifficulty [32]byte
+}
+
+func NewScryptParams(n, r, p uint) ScryptParams {
+	return ScryptParams{
+		n: C.ulong(n),
```

Comment:
What's the reason for long instead of `uint32_t`?

Reply:
Actually, it should be `C.size_t`. The type on Rust side is `usize`, which is "pointer type size". Fixed.